### PR TITLE
Add order status component

### PR DESCRIPTION
**Pull request by mmarkusik**
I want to merge this change because it adds order status to order details page. Also added the new statusChip component, according to [this figma file](https://www.figma.com/file/ZSAaKFWi05Weyuyy7GsAR4/WIP-BRANDING-SALEOR?node-id=754%3A2887).

https://app.clickup.com/t/2549495/SALEOR-1393

**PR intended to be tested with API branch:** feature/order-confirmation
### Pull Request Checklist

1. [ ] All visible strings are translated with proper context.
1. [ ] All data-formatting is locale-aware (dates, numbers, and so on).
1. [ ] Translated strings are extracted.
1. [ ] Number of API calls is optimized.
1. [ ] The changes are tested.
1. [ ] Data-test are added for new elements.
1. [ ] Type definitions are up to date.
1. [ ] Changes are mentioned in the changelog.

### Test environment config

API_URI=https://master.staging.saleor.rocks/graphql/

### CODE REVIEW

**Comment by patrys** [comment]
Task linked: [SALEOR-1393 Show unconfirmed order badge on order details](https://app.clickup.com/t/2549495/SALEOR-1393)

File: src/components/StatusChip/StatusChip.tsx [hunk 1]
```diff
@@ -0,0 +1,75 @@
+import { makeStyles } from "@material-ui/core/styles";
```
Comment (krzysztofwolski) [inline src/components/StatusChip/StatusChip.tsx:1]: Missing storybook stories

**Review by krzysztofwolski — CHANGES_REQUESTED** [review]

**Review by orzechdev — APPROVED** [review]

File: src/types/globalTypes.ts [hunk 2]
```diff
@@ -649,6 +649,13 @@ export enum OrderStatus {
   UNFULFILLED = "UNFULFILLED",
 }
 
+export enum OrderStatusType {
+  error = "error",
+  neutral = "neutral",
+  alert = "alert",
+  success = "success"
```
Comment (krzysztofwolski) [inline src/types/globalTypes.ts:656]: Lower case enum was generated by codegen or added by hand?

Comment (mmarkusik) [reply src/types/globalTypes.ts:656]: By hand, change to uppercase?

Comment (krzysztofwolski) [reply src/types/globalTypes.ts:656]: @mmarkusik file is build based on graphql schema file. If you want to add new enums, add it there and then run `npm run build-types`.

**Review by krzysztofwolski — APPROVED** [review]

**Pull request merged**